src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -166,6 +166,21 @@ private void innerTestGetPath(String s) throws IOException {
         Assert.assertTrue(size>0);
     }
 
+    @DataProvider
+    public Object[][] absoluteNames() {
+        return new Object[][] {
+                {"/local/example.txt", "/local/example.txt"},
+                {"/local/file://example.txt", "/local/file:/example.txt"},
```

Comment:
This is a strange one... can you explain why `"/local/file://example.txt"` becomes `"/local/file:/example.txt"`?

Reply:
Due to normalization of the path (removal of redundant `/`) - I added a comment here to clarify for the future.